libs/SoloPlay/Functions/MiscOverrides.js

```diff
@@ -486,8 +486,8 @@ Misc.checkSocketables = function () {
 			}
 
 			break;
-		case sdk.itemquality.Set:
-		case sdk.itemquality.Unique:
+		case item.set:
```

Comment:
This case structure needs to be reverted back, it's checking the numerical value instead of the boolean one

Reply:
done